spec/rubocop/cop/rspec/describe_class_spec.rb

```diff
@@ -51,6 +51,33 @@
     RUBY
   end
 
+  context 'with non-class constants' do
```

Comment:
WDYT of "with class or constant name as a string literal"?

Reply:
> WDYT of "with class or constant name as a string literal"?

OK, but "class or constant" is confusing (class is constant, is not it?), so I've left "with constant name as a String literal".

Reply:
It turned into "when argument is a String literal".